opts/opts.go

```diff
@@ -277,7 +283,7 @@ func ValidateExtraHost(val string) (string, error) {
 }
 
 func ValidateLabel(val string) (string, error) {
-	if strings.Count(val, "=") != 1 {
+	if strings.Count(val, "=") < 1 {
 		return "", fmt.Errorf("bad attribute format: %s", val)
 	}
```

Comment:
@thaJeztah Ok so it's not the same regexp as an environment variable. I think it makes more sense to not include this restriction here for the ``LABEL`` then.